tools/build.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 # Compile the libtimidity C codebase to JavaScript with emscripten
-BUILD_FLAGS="-s STRICT=1 -s ALLOW_MEMORY_GROWTH=1 -s MODULARIZE=1 -s EXPORT_NAME=LibTimidity -s EXPORTED_FUNCTIONS=@tools/exports.json -s EXTRA_EXPORTED_RUNTIME_METHODS=@tools/exports-runtime.json"
+BUILD_FLAGS="-s STRICT=1 -s ALLOW_MEMORY_GROWTH=1 -s MODULARIZE=1 -s EXPORT_NAME=LibTimidity -s EXPORTED_FUNCTIONS=@tools/exports.json -s EXTRA_EXPORTED_RUNTIME_METHODS=@tools/exports-runtime.json -s FORCE_FILESYSTEM=1"
```

Comment:
No, I don't think it was, I forgot to remove it. I'll remove it and verify it still works.

Reply:
> Can you share what version of `emcc` you're using? I just want to make sure I'm using a version at least as new as yours.

2.0.7

> Also – can you remove the `libtimidity.js` and `libtimidity.wasm` files as I want to generate them myself since it's impossible to audit binary output files like these :)

Sure. I'll update this PR later this evening.

> > Also, via debugging in the browser I realized that my midi files had empty tracks, which broke the parser. I was able to trim the excess tracks and everything worked great.
> 
> I don't see the changes for this in the PR. Did you mean to include these?

Oh, I didn't mean any change here. I was programmatically making midi files, and accidentally was generating them with some empty tracks. I didn't debug why the parser failed for an empty track–it was simpler to fix my files. Probably can just ignore, I figure empty tracks aren't common.

Reply:
Excellent. 2.0.7 is the latest version on Homebrew.